kuma/static/js/wiki-edit.js

```diff
@@ -396,8 +396,9 @@
     // Injects a DIV with language to the effect of "you had a previous draft, want to restore it?"
     // This takes the place of an ugly, ugly confirmation box :(
     var $draftDiv;
-    function displayDraftBox(content) {
-        var text = gettext('You have a draft in progress. <a href="" class="restoreLink">Restore the draft content</a> or <a href="" class="discardLink">discard the draft</a>.');
+    function displayDraftBox(content, draft_time) {
+        var draft_time_str = draft_time ? ' (' + draft_time + ')' : '';
+        var text = gettext('You have a draft in progress'+ draft_time_str +'. <a href="" class="restoreLink">Restore the draft content</a> or <a href="" class="discardLink">discard the draft</a>.');
```

Comment:
I'm pretty sure that this will break gettext since it suddenly has a variable. Not sure how to do that instead though.